src/OpenLoco/Company.h

```diff
@@ -113,7 +146,8 @@ namespace OpenLoco
         int16_t observationY;                // 0x8BC0;
         uint16_t observationObject;          // 0x8BC2;
         uint16_t var_8BC4;
-        uint8_t pad_8BC6[0x8BCE - 0x8BC6];
+        OwnerStatus ownerStatus; // 0x8BC6
+        uint8_t pad_8BC6[0x8BCE - 0x8BCA];
```

Comment:
just spotted this today this should say `pad_8BCA`

Reply:
Good catch! Fixed.